src/js/demoGame.js

```diff
@@ -0,0 +1,22 @@
+import { Calc } from './calc';
```

Comment:
please choose and follow one naming convention for files

Reply:
fixed